templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -292,12 +292,10 @@
 
 		<div class="ui participants">
```

Comment:
`.ui.participants`  still is referenced here?

Reply:
I've refactored it to remove this wrapper, but I guess it makes sense to keep the `participants` class for consistency reason only as the other lists also all have such a name:

<img width="341" alt="image" src="https://user-images.githubusercontent.com/115237/101417289-c34cea80-38eb-11eb-98a6-2dc9d463737a.png">

Reply:
Actually, decided to remove it.